docs/Deploying/Configuration.md

```diff
@@ -2582,31 +2582,19 @@ packaging:
 
 > MDM features are not ready for production and are currently in beta. These features are disabled by default. To enable these features set `FLEET_DEV_MDM_ENABLED=1` as an environment variable.
 
-#### dev_mdm_enabled
-
-This is the first of two feature flags required to turn on MDM features. This feature flag should be set to `1` before you follow the [setup MDM instructions](../Using-Fleet/Mobile-device-management.md#set-up).
-
-- Default value: ""
-- Environment variable: `FLEET_DEV_MDM_ENABLE`
-- Config file format:
-  ```
-  dev:
-    mdm_enable: 1
-  ```
-
```

Comment:
@mna @lukeheath do you know why we removed this section for `FLEET_DEV_MDM_ENABLE` out? 

My understanding is that this flag is required for the beta. Is this documented elsewhere?

Reply:
@noahtalerman @lukeheath It's documented just above (line 2583):

> To enable these features set `FLEET_DEV_MDM_ENABLED=1` as an environment variable.

The thing is that this is **only** an environment variable, not a command-line flag, which is why I removed the section that describes it as a flag that can also be provided as yaml.